themesh.html

```diff
@@ -50,30 +50,40 @@
 </head>
 
 <body>
-  <header role="banner" id="fh5co-header">
-    <div class="fluid-container">
-      <nav class="navbar navbar-default navbar-fixed-top js-fullheight">
-        <div id="navbar" class="navbar-collapse js-fullheight">
-          <ul class="nav navbar-nav navbar-left"></ul>
-        </div>
-      </nav>
-    </div>
-  </header>
-
-  <section id="fh5co-home" style="background-image: url(images/full_image_1.jpg)" data-stellar-background-ratio="0.5">
+  <!-- MAIN HEADER SECTION -->
+  <section id="fh5co-home" data-section="home">
     <div class="gradient"></div>
     <div class="container">
-      <div class="text-left">
-        <a href="./index.html">
-          <i class="active icon-arrow-circle-left"></i> Home</a>
+      <div class="text-wrap">
+        <div class="text-inner">
+          <div class="row">
+            <div class="sticky-top">
+              <a href="./index.html"> <i class="active icon-arrow-circle-left"></i> Home</a>
+            </div>
+            <div class="col-md-8 col-md-offset-2 text-center">
+              <h1 class="to-animate">dltx.io</h1>
+              <hr class="to-animate mb-5">
+              <h1 class="to-animate pt-5">The Mesh</h1>
+            </div>
+          </div>
+        </div>
       </div>
-      <div class="col-md-8 col-md-offset-2 text-center">
-        <h1 class="to-animate">dltx.io</h1>
-        <hr class="to-animate" />
-        <h2 class="to-animate">THE MESH</h2>
-        <p class="to-animate">The DLTx mesh a <a href="https://www.holacracy.org/explore">holacracy</a> based organisation, which is a system for managing a company where there are no assigned roles and employees have the flexibility to take on various tasks and move between teams freely. The organizational structure of a holacracy is rather flat, with there being little hierarchy.</p>
+    </div>
+  </section>
+
+  <section id="fh5co-services" data-section="services">
+    <div class="fh5co-services">
+      <div class="container">
+        <div class="row">
+          <!-- HERO SECTION PHILOSOPHY -->
+          <div class="col-md-12 section-heading text-center">
+            <h3 class="to-animate">The DLTx mesh a <a href="https://www.holacracy.org/explore">holacracy</a> based organisation, which is a system for managing a company where there are no assigned roles and employees have the flexibility to take on various tasks and move between teams freely. The organizational structure of a holacracy is rather flat, with there being little hierarchy.</h3>
```

Comment:
That sentence doesn't read properly. What is the point you are trying to get across?